### PR TITLE
Short slideset to introduce CodeRefinery

### PR DESCRIPTION
**Pull request by samumantha**
Rendered as slides: https://cicero.xyz/v3/remark/0.14.0/github.com/coderefinery/presentations/basic/2024_CodeRefinery_basic.md

- Useful for anyone who may want to attach that to their presentation
- Can be shared with ambassadors
- Will also be made available as ppt once content is decided :) 
- And can then also be turned into informative poster.

Please comment/edit on what you think should be in this presentation.

### CODE REVIEW

**Comment by bast** [comment]
Some notes for improvements:
- Ambassador program on last slide
- I will think about how to improve lessons slide (too much text but does not show all we have; I will think how to present it better)

**Comment by samumantha** [comment]
Good points! Will also think. 
Maybe some graphic similar to Heidi's 6 steps but with our lessons and related pictures?

**Comment by bast** [comment]
Yes. Something graphical, can be hand-drawn, that gives an overview. The message could be: we already have a lot but there is room for more.

**Comment by samumantha** [comment]
WIP, comments and ideas welcome :)

**Comment by bast** [comment]
This is really brilliant.

**Comment by samumantha** [comment]
I'll still rearrange the lessons to spread them more, any other comments now also welcome then I can still fix

**Comment by samumantha** [comment]
- [x] remove 2025 mention (not relevant)
- [x] add instead number of people trained / year -> about 500/year
- [x] link to lessons in slide 1
- [ ] "teaching essential skills besides coding" , "train you in FAIR research software development", something to speak to researchers
- [x] link to youtube instead of youtube
- [x] add slide targeted add participants: https://coderefinery.org/blog/bring-your-own-classroom/
- [ ] Main points: difficulties in computing, you are not alone; CR as crashcourse you always wanted for scientific software development; come together and collaborate

**Comment by samumantha** [comment]
Don't know how to resolve those last two open points. Any ideas? 
Or should we merge like this?

**Comment by bast** [comment]
Should we replace "Join our next workshop September 2024" by "Join our next workshop" and link to upcoming workshops? To make the slide deck not be outdated by October 2024.

**Comment by bast** [comment]
> * "train you in FAIR research software development"

The very first slide could have CC-BY logo, a small text with version ("version Aug 2024") and then above the small text a larger "We train you in FAIR research software development".

Let me know if I should commit the above suggestions.

**Comment by samumantha** [comment]
Very good points! Yes please edit

**Review by bast — APPROVED** [review]
Ready to go! Thank you!

**Comment by bast** [comment]
TLDR side preview: https://cicero.xyz/v3/remark/0.14.0/github.com/coderefinery/presentations/basic/2024_CodeRefinery_basic.md

**Comment by samumantha** [comment]
Turning this now also into odp format, then we can send it out to ambassadors
